test/intersections.jl

```diff
@@ -150,6 +150,34 @@
     @inferred someornone(s1, s2)
   end
 
+  @testset "Rays" begin
```

Comment:
tests for 2D
![image](https://user-images.githubusercontent.com/67096719/169515421-96968574-5b52-48a6-8cb0-b73cb4b6c9a8.png)

Please give feedback before adding tests in 3D

What else has to be tested?

Reply:
This is a great set of rays to test 👍🏽